### PR TITLE
Lit17149 life

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT17149Life.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT14519Life" xml:lang="en" type="work">
```
Comment (CarstenHoffmannMarburg) [inline new/LIT17149Life.xml:4]: xml:id must correspond to file name.

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

**Comment by MarcinKrawczuk** [comment]
I was instructed to simply rename the file (I have corrected the xml:id),
so the proper file name is LIT7149Life

wt., 19 lis 2024 o 20:33 Carsten Hoffmann ***@***.***>
napisał(a):

**Comment by eu-genia** [comment]
the file name should indeed be LIT7149Life. careful now it is 17149 instead of 7149

the file and the xml:id should be the same, 7149

**Comment by MarcinKrawczuk** [comment]
Oh, now I see that the file name was wrong in my GitHub folder... Sorry
about that. I have renamed it.

wt., 19 lis 2024 o 21:30 Eugenia ***@***.***> napisał(a):

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**